AnkiDroid/src/main/java/com/ichi2/anki/CardBrowser.java

```diff
@@ -758,17 +749,35 @@ public void onNothingSelected(AdapterView<?> parent) {
 
         getWindow().setSoftInputMode(WindowManager.LayoutParams.SOFT_INPUT_STATE_ALWAYS_HIDDEN);
 
+        long deckId = getCol().getDecks().selected();
+        mDeckSpinnerSelection = new DeckSpinnerSelection(CardBrowser.this, R.id.toolbar_spinner, false);
```

Comment:
We use `CardBrowser.this` if we're in a non-static class to refer to the outer class, not necessary here.

```suggestion
        mDeckSpinnerSelection = new DeckSpinnerSelection(this, R.id.toolbar_spinner, false);
```